messages/root/messages.go

```diff
@@ -1,16 +1,16 @@
 package root
 
 var (
-	RootUsage = `azion <command> <subcommand> [flags]
-azion [flags]
- 	`
+	RootUsage       = "azion <command> <subcommand> [flags]\n azion [flags]\n\t"
 	RootDescription = "Azion CLI %s"
 	RootHelpFlag    = "Displays more information about the Azion CLI"
 	RootDoNotUpdate = "Do not receive update notification"
 	RootLogDebug    = "Displays log at a debug level"
-	RootLogQuiet    = "Silences log completely; mostly used for automation purposes"
+	RootLogLevel    = "Set the logging level, \"debug\", \"info\", or \"error\". Default \"info\"."
+	RootLogSilent   = "Silences log completely; mostly used for automation purposes"
 	RootTokenFlag   = "Saves a given personal token locally to authorize CLI commands"
 	RootConfigFlag  = "Sets the Azion configuration folder for the current command only, without changing persistent settings."
+	RootYesFlag     = "yes global that says yes to everything"
```

Comment:
Answers all yes/no questions with a yes
@gabriel-azion what do you think?

Reply:
```suggestion
	RootYesFlag     = "Answers all yes/no interactions automatically with yes"
```